hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -29,10 +29,10 @@
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.ExecutorCompletionService;
 import java.util.concurrent.ThreadPoolExecutor;
-import java.util.concurrent.Callable;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 
+import org.apache.hadoop.fs.statistics.StreamStatisticNames;
```

Comment:
nit: needs to go into the real org.apache block. The IDE is getting confused because of the third party stuff. as this is a "real" hadoop import it should go down below; when we backport the google ones will go back to their unshaded names

Reply:
Sorry, my bad, should've double-checked.